Modules/System/Rest Client/src/HttpMethod.Enum.al

```diff
@@ -0,0 +1,68 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+/// <summary>
+/// This enum contains the REST Http Methods.
+/// </summary>
+enum 2350 "Http Method"
+{
+    Extensible = true;
```

Comment:
Is this extensibility of this ENUM really intended?

Reply:
Not really. The enum was copied from the OAuth module, and I kept everything the same.
But thinking about it, for REST you use 5 HTTP methods: GET, POST, PATCH, PUT, DELETE
That also means that the HEAD and OPTIONS methods can be removed. Would you agree?

When the module gets its own namespace, then another module could define a Http Method enum in its own namespace in case other HTTP methods are needed.